deploy/olm-catalog/argocd-operator/0.14.0/argoproj.io_argocds.yaml

```diff
@@ -446,6 +446,40 @@ spec:
                         type: object
                     type: object
                 type: object
+              argoCDAgent:
+                description: ArgoCDAgent defines configurations for the ArgoCD Agent
+                  component.
+                properties:
+                  principal:
+                    description: Principal defines configurations for the Principal
```

Comment:
One more thing, I think possibly these `v0.14.0` version of the files shouldn't be modified? You can try reverting the changes and running `make bundle` etc, again, to see if the files are regenerated.